CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## edge-23.2.1
+
+This edge release sees the `linkerd-cni` plugin moved to
+`linkerd2-proxy-init` and released from that repository. An iptables
+improvement to `linkerd-cni` and `proxy-init` is the main focus. Other
+minor fixes are also included.
+
+* Upgraded `linkerd-cni` to v1.0.0 which is now published from
+  `linkerd2-proxy-init` and makes iptables rules idempotent (thanks @jim-minter!)
+  as well as minor logging improvements.
+* Upgraded `proxy-init` from v2.1.0 to v2.2.0 which makes `iptables` rules
+  idempotent (thanks @jim-minter!).
```

Comment:
Both of these points are sound to me but perhaps we could separate them based on the change, like so:

```suggestion
* Improved logging in `proxy-init` and `linkerd-cni`
* Changed `proxy-init` iptables rules to be idempotent upon init pod restart (thanks @jim-minter!)
```

I would maybe not specify that the version has been bumped. We generally don't do that when we release the proxy and for consistency it might be worth applying the same rationale here. If the team or you think we should include it though, we could add another line to the suggestion I gave above

```
* Bumped `proxy-init` to v2.2.0 and `linkerd-cni` v1.0.0
```

Reply:
`Changed `proxy-init` iptables rules to be idempotent upon init pod restart (thanks @jim-minter!)`

I think the challenge with this comment is customers might miss that it affects `linkerd-cni` as well.

Reply:
Update looks great 👍🏻